token-contracts/bnUSD/build.gradle

```
@@ -0,0 +1,28 @@
+plugins {
+    id 'java'
+}
+
+version '1.0.0'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compileOnly 'foundation.icon:javaee-api:0.9.1'
+    testImplementation 'foundation.icon:javaee-unittest:0.9.2'
+
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.8.1'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.8.1'
+}
+
+test {
+    useJUnitPlatform()
+}
```

```suggestion
test {
    useJUnitPlatform()
    finalizedBy jacocoTestReport 
}

jacocoTestReport {
    dependsOn test
    reports {
        xml.required = false
        csv.required = false
        html.outputLocation = layout.buildDirectory.dir('jacocoHtml')
    }
}
```

Add this and then to project gradle add 
`apply plugin: 'jacoco'`
and you can get coverage reports of your unitTests. You wont need a 100% but you can look through there to check you are covering all the good spots :slightly_smiling_face: